colormath/__init__.py

```diff
@@ -1 +1,3 @@
+#!/usr/bin/env python
```

Comment:
Hi @rec!

I brought this line along with the encoding one. I actually needed and still need to launch quickly any of the modules, i.e. good when are introspecting a codebase. You don't need a `if __name__ == '__main':` clause to run content of any Python module and on Windows you don't need to make the script executable at all, see PEP 397.

The fact the line is here does not really hurt and makes some things easier for me.

Reply:
Definitely understand how it may help your particular workflow, but this is a bit of an anti-pattern. We should probably be optimizing worfkflows around the test suite.